### DISCUSSION TITLE
If I want to save information in a database how do I do that?

### DISCUSSION THREAD

**Discussion by Sk8b04rd3r** [community/community#69931] [New to GitHub] [answered]
I have a Mac. Im learning python and my first project will be using flask. I want to save information permanently. How can I do that?

**Comment by fredschuck** [comment]
Before we can answer, do you have a planned database system to use? (i.e., PostgreSQL, MongoDB, MySQL)?

**Reply by Sk8b04rd3r** [reply]
No, I don't have that :/ Ok. I will read up on that first then. Thank you!

**Reply by fredschuck** [reply, marked as answer]
So, I have used all three. I personally have most experience with PostgreSQL and Flask. 

Here is a little course on Postgres.
https://www.youtube.com/watch?v=qw--VYLpxG4

Hopefully that helps! Checkout my [project](https://github.com/fredschuck/workout_app). It's relatively simple, but hopefully it gives you an idea on file structure and some things to look for. I highly suggest looking for other flask apps to see

**Reply by Sk8b04rd3r** [reply]
Thank you!

**Comment by fredschuck** [comment]
BTW, this belongs in Programming Help, not New to Github :)

**Comment by thevindu-w** [comment]
You can save them in a database like MySQL. You can refer to this answer at [https://stackoverflow.com/a/622308](https://stackoverflow.com/a/622308) for an example. There are many tutorials on connecting Python with databases.

**Reply by Sk8b04rd3r** [reply]
Thank you!

**Comment by RomanchenkoAS** [comment]
Choosing data storage method does not depend on OS. PostgreSQL is a good choice if you are using ORM-system, but MongoDB would be much easier for a beginner to get hang of.
